xml/cl.xml

```diff
@@ -1714,10 +1813,16 @@ server's OpenCL/api-docs repository.
         <enum value="0x2030"        name="CL_CONTEXT_MEMORY_INITIALIZE_KHR"/>
         <enum value="0x2031"        name="CL_DEVICE_TERMINATE_CAPABILITY_KHR"/>
         <enum value="0x2032"        name="CL_CONTEXT_TERMINATE_KHR"/>
+    </enums>
+
+    <enums start="0x2033" end="0x2034" name="cl_kernel_sub_group_info" vendor="Khronos">
         <enum value="0x2033"        name="CL_KERNEL_MAX_SUB_GROUP_SIZE_FOR_NDRANGE"/>
         <enum value="0x2033"        name="CL_KERNEL_MAX_SUB_GROUP_SIZE_FOR_NDRANGE_KHR"/>
         <enum value="0x2034"        name="CL_KERNEL_SUB_GROUP_COUNT_FOR_NDRANGE"/>
         <enum value="0x2034"        name="CL_KERNEL_SUB_GROUP_COUNT_FOR_NDRANGE_KHR"/>
+    </enums>
+
+    <enums start="0x2035" end="0x201F" name="enums.2035" vendor="Khronos" comment="Reserved for interop with other APIs">
```

Comment:
This is no worse than it was previously, but not all of these enums are used for "interop with other APIs".

I'm fine addressing this with a subsequent PR if you'd like, but please file an issue referencing this comment so we don't forget about it.  Thanks!

Reply:
the comment is obviously wrong changing _everything_ at once is usually not the best way to get PRs accepted :)

Reply:
This is just historical hangover - when the block was initially reserved that was its purpose. Sometimes it's useful to actually keep related enums contiguous and in those cases actually reserving a block specific to that purpose makes sense, but it doesn't here and now.